src/Microsoft.AspNet.Mvc.Common/TypeExtensions.cs

```diff
@@ -117,5 +119,24 @@ public static Type[] GetTypeArgumentsIfMatch([NotNull] Type closedType, Type mat
             var openType = closedType.GetGenericTypeDefinition();
             return (matchingOpenType == openType) ? closedTypeInfo.GenericTypeArguments : null;
         }
+
```

Comment:
Ignore changes to this file, they are temporary for my convenience and will be removed before pushing